tests/overlay_tests.cpp

```diff
@@ -518,6 +517,7 @@ TEST_F(OverlayTest, checkMasterAgentComm)
 
   AgentInfo masterAgentInfo;
   masterAgentInfo.CopyFrom(state->agents(0));
+  info->clear_configuration_attempts();
```

Comment:
I think we need this only because you are setting a default? If we don't have the default in the protobuf we shouldn't need this?

Reply:
In that test case we checks that agent and master modules have the same configuration. Only agent module has `configuration_attempts` value, so we have to clear it to be able to compare agent module response with master module response.

Reply:
👍